content/terminal/custom-backend/custom-backend.md

```diff
@@ -32,54 +36,145 @@ import TutorialVideo from '@site/src/components/General/TutorialVideo.tsx';
 />
 */}
 
-This is a versatile way to connect any data to the OpenBB Terminal. Whether your API is hosted internally or externally, this method provides a standardized structure that OpenBB Terminal Pro widgets can render.
+## Widget Types
+
+OpenBB Terminal Pro supports various ways to visualize data through widgets. Each method offers unique features and can be configured to suit different data presentation needs. Below are the different visualization methods available:
+
+1. **Table Widget**:
+   - Provides a structured view of data in rows and columns.
+   - Ideal for displaying detailed datasets.
+   - Can be configured to display charts with Native Charting.
+   - [Learn more about Table Widgets](Widgets/table/table.md)
+
+<img className="pro-border-gradient" width="600" alt="table" src="https://openbb-assets.s3.us-east-1.amazonaws.com/docs/pro/simple-table.png" />
+
+
+2. **Chart Widgets**:
+   - Offers graphical representation of data.
+   - Supports Plotly charts and can be configured to display different types of charts. Support for more charting frameworks will be coming soon.
+   - [Learn more about Chart Widgets](Widgets/chart.md)
+
+<img className="pro-border-gradient" width="600" alt="chart" src="https://openbb-assets.s3.us-east-1.amazonaws.com/docs/pro/plotly-chart.png" />
 
-1. **Design and implement your API**: Choose your preferred programming language and framework. Ensure that the API can return data in JSON format, which is required for widget integration.
+3. **Markdown Widget**:
+   - Displays formatted text and images.
+   - Can be used to display markdown, txt, and docx files.
+   - [Learn more about Markdown Widgets](Widgets/markdown.md)
+
+<img className="pro-border-gradient" width="600" alt="markdown" src="https://openbb-assets.s3.us-east-1.amazonaws.com/docs/pro/markdown-widget.png" />
+
+4. **Metric Widget**:
+   - Displays a single value or metric.
+   - Useful for highlighting key performance indicators.
+   - [Learn more about Metric Widgets](Widgets/metric.md)
+
+<img className="pro-border-gradient" width="600" alt="metric" src="https://openbb-assets.s3.us-east-1.amazonaws.com/docs/pro/metric-widget.png" />
+
+
+## Design and implement your API
```

Comment:
At this point - would it make more sense to just move onto a quickstart page separately.  We should also have a link to the examples.